### DISCUSSION TITLE
how to work with prisma ?

### DISCUSSION THREAD

**Discussion by MeetChaudhari** [neutralinojs/neutralino.js#75] [Q&A]

**Comment by burnoutprojects** [comment]
Hi, if I understand correctly your answer, you have to use a server as the backend. Please check this two implementions of [Bun](https://github.com/hschneider/neutralino-ext-bun) and [Node.js](https://github.com/hschneider/neutralino-ext-node) for JavaScript/Typescript implementions and this for [Python](https://github.com/hschneider/neutralino-ext-python) implementions
